lib/css_parser/rule_set.rb

```diff
@@ -107,15 +111,16 @@ def each_declaration # :yields: property, value, is_important
     # TODO: Clean-up regexp doesn't seem to work
     #++
     def declarations_to_s(options = {})
-     options = {:force_important => false}.merge(options)
-     str = String.new
-     each_declaration do |prop, val, is_important|
-       importance = (options[:force_important] || is_important) ? ' !important' : ''
-       str << "#{prop}: #{val}#{importance}; "
-     end
-     str.gsub!(/^[\s^(\{)]+|[\n\r\f\t]*|[\s]+$/mx, '')
-     str.strip!
-     str
+      options = options.dup
+      options[:force_important] ||= false
```

Comment:
can we just remove the dup + `||=` ... force_important seems to do nothing since it's falsy when not set anyway ?